contracts/concordium-governance-committee-election/Cargo.toml

```diff
@@ -12,7 +12,6 @@ description = "A smart contract for facilitating Concordium goverance committee
 default = ["std"]
 std = ["concordium-std/std"]
 wee_alloc = ["concordium-std/wee_alloc"]
-serde = ["dep:serde"]
```

Comment:
Shouldn't the optional dependency also be removed?

Reply:
good catch